src/java2yaml/Steps/GenerateDocument.cs

```diff
@@ -13,7 +13,22 @@ public Task RunAsync(ConfigModel config)
         {
             return Task.Run(() =>
             {
-                //TODO
+                foreach (var path in config.RepositoryFolders)
+                {
+                    var procedure = new StepCollection(
+                        new RestoreDependency(path),
+                        new RunJavadoc(path));
+
+                    try
+                    {
+                        procedure.RunAsync(config).Wait();

```

Comment:
Usually, it recommends to use `await xxx` over `xxx.Wait()`

Reply:
changed to `await `